config/samples/run_v1alpha3_clusterbootstrap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: run.tanzu.vmware.com/v1alpha3
+kind: ClusterBootstrap
+metadata:
+  name: clusterbootstrap-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
Add sample values for `spec`

Reply:
Could we remove this sample if not used anywhere? Same comment for below.